src/components/Identity/createComponent.js

```diff
@@ -30,6 +30,8 @@ export default ({
             setLegacyEcid(ecid);
           }
         }
+        // For sendBeacon requests, getEdge() will return {}, so we are using assign here
+        // so that sendBeacon requests don't override the edge info from before.
         edge = assign(edge, response.getEdge());
```

Comment:
I don't care if you change this, but `edge = ` is unnecessary. `assign(edge, response.getEdge());` will suffice.